### DISCUSSION TITLE
how to set GID and UID for the docker container?

### DISCUSSION THREAD

**Discussion by unlucio** [datawhores/OF-Scraper#371] [Q&A]
Docker containers are brilliant and I'm glad this tool has that option.
I'm running in a bit of a problem tho because the container uses:
```
ofscraper@3f73d773ad93:/download$ id
uid=1000(ofscraper) gid=1000(ofscraper) groups=1000(ofscraper)
```

and, due to the destination where it has to be written, I'd like for it to use 501.
I tried with `UID=${UID} GID=${GID} docker-compose run scraper` but it still ended up using user 1000.

**Comment by datawhores** [comment]
https://github.com/boxboat/fixuid
See
Set default values inside of compose

**Comment by datawhores** [comment]
> Docker containers are brilliant and I'm glad this tool has that option. I'm running in a bit of a problem tho because the container uses:
> 
> ```
> ofscraper@3f73d773ad93:/download$ id
> uid=1000(ofscraper) gid=1000(ofscraper) groups=1000(ofscraper)
> ```
> 
> and, due to the destination where it has to be written, I'd like for it to use 501. I tried with `UID=${UID} GID=${GID} docker-compose run scraper` but it still ended up using user 1000.

I planning on looking into this again
we might need another solution for managing permissions
